src/https_client.c

```diff
@@ -50,6 +50,11 @@ static void https_fetch_ctx_init(https_client_t *client,
   client->fetches = ctx;
 
   CURLcode res;
+  
+  //set new ca path
+  curl_easy_setopt(ctx->curl, CURLOPT_CAPATH, "/mnt/jffs2/app/doh/etc/ssl/certs/");
```

Comment:
Hard-coding a CA auth path here is not a good idea. You should install a version of curl with the right defaults and put your certs in the correct path.